src/test/rpc/RPCCall_test.cpp

```diff
@@ -6403,7 +6403,7 @@ std::string
 updateAPIVersionString(const char* const req)
 {
     static std::string version_str =
-        std::to_string(RPC::apiMaximumSupportedVersion);
+        std::to_string(RPC::apiMinimumSupportedVersion);
     static auto place_holder = "%MAX_API_VER%";
```

Comment:
If we are putting minimum version as `version_str`. Should this line instead be `static auto place_holder = "%MIN_API_VER%";` ?

Reply:
I changed it back to `RPC::apiMaximumSupportedVersion` and we will see what we eventually decide to use as default. Could be `apiVersionIfUnspecified` as you said earlier.